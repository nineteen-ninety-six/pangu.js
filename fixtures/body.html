<div id="p1">
    <h1>好abc個秋</h1>
    <div>
        <p><a href="#">abc1</a>漢字一</p>
        <p><a href="#">abc2</a><a href="#">漢字二</a></p>
    </div>
    <span class="TH_index">作者</span><span>losel ( )</span><br><span class="TH_index">標題</span>
    <div>
        <p><!-- comment -->123漢字abc</p>
        <p>漢字abc</p>
        <br>
        <p>漢字123</p>
        <a target="_blank" render="ext" extra-data="type=topic" class="a_topic" href="#">#去新西兰逛超市#</a>再也不用担心
    </div>
    <p>機動戰士GUNDAM逆襲的夏亞<a href="#"><s><label></label>+</s><b>炸裂！</b></a></p>
    <p>陳上進 likes <a href="#">林依諾</a>'s status.</p>
    <pre><div>請問Jackey的鼻子有幾個？</div></pre>
    <div>
        <p>123漢字abc</p>
        <p>123漢字<br>abc</p>
        <br>
        <textarea>中文abc漢字5566</textarea>
        <p>中文7788</p>
        <textarea><div>中文abc漢字5566<span>漢字123</span></div></textarea>
        <label>選項(必填) ex</label>
        <input type="text" value="中文123漢字abc" />
        <p>中文<br>2266</p>
        <hr>
    </div>
    <a href="#" name="addcomment" class="meta-item toggle-comment"><i class="z-icon-comment"></i>15条评论</a>
    <code><span>你從什麼時候產生了Jackie的鼻子只有1個的錯覺？</span></code>
    <div>
        <div contenteditable="true">abc漢字1</div>
        <div contenteditable="false">abc漢字2</div>
        <div g_editable="true">abc漢字3</div>
        <div g_editable="false">abc漢字4</div>
    </div>
    <ul>
      <li><a href="http://vinta.ws/hello_world/">一個超連結with英文，網址是英文</a></li>
      <li><a href="http://vinta.ws/中文網址with英文.html">oh一個超連結with英文，網址包含中文</a></li>
    </ul>
    <a href="#"><em class="S_txt1">搜 “</em><span class="S_spetxt">Vinta陳上進</span><em class="S_txt1">” 相关微博</em></a>
</div>
