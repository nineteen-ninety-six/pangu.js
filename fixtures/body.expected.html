<div id="p1">
    <h1>好 abc 個秋</h1>
    <div>
        <p><a href="#">abc1</a> 漢字一</p>
        <p><a href="#">abc2</a><pangu> </pangu><a href="#">漢字二</a></p>
    </div>
    <span class="TH_index">作者</span><span> losel ( )</span><br><span class="TH_index">標題</span>
    <div>
        <p><!-- comment -->123 漢字 abc</p>
        <p>漢字 abc</p>
        <br>
        <p>漢字 123</p>
        <a target="_blank" render="ext" extra-data="type=topic" class="a_topic" href="#">#去新西兰逛超市#</a>再也不用担心
    </div>
    <p>機動戰士 GUNDAM 逆襲的夏亞 <a href="#"><s><label></label>+</s><b> 炸裂！</b></a></p>
    <p>陳上進 likes <a href="#">林依諾</a>'s status.</p>
    <pre><div>請問Jackey的鼻子有幾個？</div></pre>
    <div>
        <p>123 漢字 abc</p>
        <p>123 漢字<br>abc</p>
        <br>
        <textarea>中文abc漢字5566</textarea>
        <p>中文 7788</p>
        <textarea>&lt;div&gt;中文abc漢字5566&lt;span&gt;漢字123&lt;/span&gt;&lt;/div&gt;</textarea>
        <label>選項 (必填) ex</label>
        <input type="text" value="中文123漢字abc">
        <p>中文<br>2266</p>
        <hr>
    </div>
    <a href="#" name="addcomment" class="meta-item toggle-comment"><i class="z-icon-comment"></i>15 条评论</a>
    <code><span>你從什麼時候產生了Jackie的鼻子只有1個的錯覺？</span></code>
    <div>
        <div contenteditable="true">abc漢字1</div>
        <div contenteditable="false">abc 漢字 2</div>
        <div g_editable="true">abc漢字3</div>
        <div g_editable="false">abc 漢字 4</div>
    </div>
    <ul>
      <li><a href="http://vinta.ws/hello_world/">一個超連結 with 英文，網址是英文</a></li>
      <li><a href="http://vinta.ws/中文網址with英文.html">oh 一個超連結 with 英文，網址包含中文</a></li>
    </ul>
    <a href="#"><em class="S_txt1">搜 “</em><span class="S_spetxt">Vinta 陳上進</span><em class="S_txt1">” 相关微博</em></a>
</div>
